src/pages/index.js

```diff
@@ -103,12 +154,14 @@ profileEditButton.addEventListener("click", () => {
   editProfilePopup.open();
 });
 
-//add card popup
+/* ---------------------------- add card popup ---------------------------- */
 
 const handleAddCardFormSubmit = (inputValues) => {
-  const card = renderCard(inputValues);
-  section.addItem(card);
-  newCardPopup.close();
+  api.addNewCard(inputValues).then(() => {
```

Comment:
Needs correction: you should also start the loading before api request and stop the loading in `finally` block. Do that for every request